File: chap1/FractionAdd.py
# Ces programmes sont sous licence CeCILL-B V1.
a = int(input())
b = int(input())
c = int(input())
d = int(input())
if (b == 0) or(d == 0):
   print("Dénominateur nul interdit !")
else:
   print(a * d + c * b)
   print(b * d)

File: chap1/FractionAdd.cpp
// Ces programmes sont sous licence CeCILL-B V1.
#include <iostream>

using namespace std;

int main() 
{
   int a,b,c,d;
   cin >> a;
   cin >> b;
   cin >> c;
   cin >> d;
   if (b == 0 || d == 0)
      cout << "Dénominateur nul interdit !" << endl;
   else 
   {
      cout << a * d + c * b << endl;
      cout << b * d << endl;
   }
}
